RHEL6_7/packages/RemovedPackages/noauto_postupgrade.d/native_rpm_dep_fix.sh

```diff
@@ -99,7 +103,7 @@ for counter in {0..10}; do
     # try just limited number of loops in most to be sure that scripts ends
     # always (in reasonable time)
     # .. in case that nothing is removed (returned 0) break the loop
-    rm_native_broken_old_rpms && break
+    rm_broken_old_rpms "$RM_CUSTOM_RPM" && break
     [ $counter -eq 10 ] && {
         [ -z "$(get_broken_deps_list)" ] && {
             log_warning "Reached iteration limit but some RPMs are still broken."
```

Comment:
The log warning does not tell the user, how to resolve the situation. Add there the information about, what should user do, when it happens (re-run the script).

Reply:
it is written just line below :)